server/src/main/antlr4/org/eclipse/lsp/cobol/core/parser/CobolParser.g4

```diff
@@ -3414,3 +3491,8 @@ idms_only_words
     | UPGRADE | USAGE_MODE
     | WCC | WHITE | WITHIN | YELLOW
     ;
+
+dacoKeywords
+    : ANA | ANALIST | AVG | DESCRIPTION | DESIGNER | DOM | ENTITY | GRS
+    | ITEM | JOB | ODETTE | OWN
+    ;
```

Comment:
Is it worth deleting the entire rule?

Reply:
Actually I added as per @temanbrcom suggestion but later we came to notice, it's intersecting with other rules. Hence I commented it's inclusion in cobolwords but kept rule , as we will be needing it in future. Please confirm if I should delete it?

Reply:
Let's leave it as is right now.